src/osuTK/osuTK.NS20.csproj

```diff
@@ -17,22 +17,29 @@
   </Target>
   <ItemGroup>
     <Compile Remove="Android-obj\**" />
+    <Compile Remove="Graphics\ES31\**" />
```

Comment:
I really don't see why this is needed for NS20. It builds just fine on my machine... Not that we're going to be using 3.1, but if we're pushing this as an alternative to OpenTK, I see no reason to exclude it.

Who knows, perhaps we can do platform-specific optimisations at the framework's end based on ES31 bindings.